### DISCUSSION TITLE
Offer sort by JP name order

### DISCUSSION THREAD

**Discussion by kimd14** [lonqie/SchaleDB#76] [Ideas and Suggestions]
It would be nice to be able to sort the site's student list by JP name order when cross-referencing with the in-game student list on the JP server.

**Comment by lonqie** [comment]
To sort by the Japanese name order, you can change the site language to Japanese and then sort by "名前".

**Reply by kimd14** [reply]
That’s logical, I suppose I’m interested in sort options across localizations though?
Even if my preference for the site’s UI language differs from the game language, being able to sort by the game’s sort order would be useful.

**Reply by lonqie** [reply]
I can understand the reasoning behind the suggestion.

However, the main problem is that language-specific data is only loaded based on the language you set the website to. Japanese names aren't included in the 'English' data that is loaded. Including this information would increase data file sizes while being redundant for ~99.9% of users visiting the site with the language set to English.

I think this is a niche use case, and temporarily switching the site language to Japanese is a minor inconvenience to fulfil this need compared to the alternative.

**Reply by kimd14** [reply]
That's entirely fair.
Purely hypothetically I might suggest an opt-in to load cross-localization data for users playing across languages (maybe even to add tooltips for localized names and terms) but it's probably not worth the add to code maintenance if there isn't more interest in the feature set.
